### ISSUE TITLE
Accessibility hooks seem to fail sometimes onElectron apps

### ISSUE THREAD

**Issue by zersiax** [electron/electron#5510]
* Electron version: unknown
* Operating system: Windows 7, Windows 10
I just had a conversation with @lee-dohm about this and he suggested making an issue about this on here.
I have found similar behavior in both the newly popular Franz  application as well as the installer for the currently in beta visual studio 2016. If i'm not mistaken, both of these applications are based on the Electron shell, although I don't know what version of the shell is being used.
Where for example Visual Studio COde, also based on Electron, sends a screenreader user into the web document to properly interact with its content, the earlier mentioned programs do not do this. 
Once the web document can actually be entered, its up to the proper use of HTML to make sure accessibility works well. However, in these cases something appears to be interfering with the accessibility subsystems, rendering the entire window unreadable. To test:
- Use the open-source, free NVDA screenreader and open up Visual Studio code.
- Once you hear a bleeping noise and the UI has loaded, press Escape and tab a few times. You will hear controls being announced.
- Now open Franz, the messaging application that combines  a number of services. Do the same experiment, without pressing escape this time. You will hear NVDA only reports 'unknown'  rather than the actual controls being announced.
- if things haven't changed since last month, you could test this with the vs 2016 installer as well. After the loading window has finished loading, try tabbing through the interface again. Nothing will be announced at all.

I'm very curious, both from a user as a dev standpoint, why this is happening and what can be done to get around this problem. What would make the accessibility APIs get this confused?

**Comment by zcbenz** [comment]
I think it is probably because Franz is using an old version of Electron, we did fix some problems with accessibility tools and I believe VS Code is using the recent versions that include the fixes.

As for Visual Studio 2016 (and its installer), it is actually not based on Electron.

**Comment by paulcbetts** [comment]
> As for Visual Studio 2016 (and its installer), it is actually not based on Electron.

VS's new Installer actually _is_ based on Electron, true story!

**Comment by zcbenz** [comment]
> VS's new Installer actually is based on Electron, true story!

Wowwwww, this is amazing!

**Comment by zersiax** [comment]
Hahaha amazing as it may be, it still has the same accessibility
issues Franz has. Even more so in fact because nothing at all is
reported. Is this also due to them using an old version?

**Comment by zcbenz** [comment]
Yeah I just checked that VS's new installer is still using 0.34.x.
